bus_sdk/publishers/event_publisher.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from bus_sdk.messages import Message
+
+
+class EventPublisher(ABC):
+    @property
+    @abstractmethod
+    def name(self) -> str:
+        raise NotImplementedError
+
+    @abstractmethod
+    def publish(self, message: Message) -> str:
```

Comment:
It's interchanging 'message' and 'event', better to always refer to the latter?

Reply:
Change `Message` to be `Event`? e.g. `ProfileEvent`...

Reply:
`Message` is now `Event` across the package.

Reply:
@thewilkybarkid I will wait until this PR is merged before integration on Profiles, slightly fewer changes if done this way round.